sys/net/gnrc/network_layer/ipv6/ext/gnrc_ipv6_ext.c

```diff
@@ -28,6 +30,124 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+/**
+ * @brief   Checks if @p protnum is a duplicate @ref PROTNUM_IPV6_EXT_HOPOPT.
+ *
+ * If @p protnum is @ref PROTNUM_IPV6_EXT_HOPOPT, the function sends a parameter
+ * problem message (if @ref net_gnrc_icmpv6_error is available), releases the
+ * packet with error `EINVAL`, and returns `false`.
+ *
+ * @note    This function should only be called *after* the first hop-by-hop
+ *          option header in the packet was parsed (if it exists) or if the
+ *          first extension header wasn't a hop-by-hop option header.
+ *
+ * @param[in] pkt       A packet in receive order to be checked with the current
+ *                      extension header (of type @p protnum) as its payload.
+ * @param[in] protnum   The protocol number of the header ingnrc_pktsnip_t::data
+ *                      of @p pkt.
+ *
+ * @return  true, when @p protnum != @ref PROTNUM_IPV6_EXT_HOPOPT.
```

Comment:
AFAIS, the function return `true` when `protnum == PROTNUM_IPV6_EXT_HOPOPT`

Reply:
Makes also more sense with the name ;-). Fixed.